tests/test_logging.py

```diff
@@ -12,8 +12,27 @@
 
 from dmtestutils.comparisons import AnyStringMatching, AnySupersetOf, RestrictedAny
 
-from dmutils.logging import init_app, RequestExtraContextFilter, JSONFormatter, CustomLogFormatter
-from dmutils.logging import LOG_FORMAT, get_json_log_format
+from dmutils.logging import init_app, JSONFormatter, CustomLogFormatter, configure_handler
+from dmutils.logging import (LOG_FORMAT, get_json_log_format, AppNameFilter, RequestExtraContextFilter,
+                             AppStackLocationFilter, AppInstanceFilter)
+
+
+def test_configure_handler(app):
+    handler = mock.Mock()
+    configure_handler(handler, app, mock.Mock())
+    filter_classes = [x[0][0].__class__ for x in handler.addFilter.call_args_list]
+    assert AppNameFilter in filter_classes
+    assert RequestExtraContextFilter in filter_classes
+    assert AppStackLocationFilter in filter_classes
+    assert AppInstanceFilter not in filter_classes
+
+
+def test_configure_handler_has_instance_index_where_present(app, os_environ):
+    os_environ.update({'CF_INSTANCE_INDEX': '1'})
+    handler = mock.Mock()
+    configure_handler(handler, app, mock.Mock())
+    filter_classes = [x[0][0].__class__ for x in handler.addFilter.call_args_list]
+    assert AppInstanceFilter in filter_classes
```

Comment:
These tests are fine, but an even better assertion style would be

```
assert filter_classes == [
    AppNameFilter,
    RequestExtraContextFilter,
    AppStackLocationFilter,
    AppInstanceFilter
]
```

So for each test, it's explicit which classes are in there (and if we remove/add any classes in future, it's very obvious).